extensions/Needinfo/template/en/default/hook/request/email-after_summary.txt.tmpl

```diff
@@ -8,6 +8,27 @@
 
 [% RETURN UNLESS flag && flag.type.name == 'needinfo' && flag.status == '?' %]
 ---
---- This request has set a needinfo flag on the [% terms.bug %].
---- You can clear it by logging in and replying in a comment.
+--- Hello from [% terms.Bugzilla %], the [% terms.bug %] tracker for Mozilla and Firefox:
+---
+--- Someone has asked for you for information about this [% terms.bug %].
+---
+--- Please go to [%+ urlbase %]show_bug.cgi?id=[% bug.bug_id %] to see the
+--- question and respond to it.
+[% IF flag.requestee.login_name == bug.reporter.login_name %]
+---
+--- Since you reported this [% terms.bug %], then the person asking the
+--- question needs more information from you to understand it.
+[% END %]
+---
+--- Responding to the question will enable developers to take further
+--- action on this [% terms.bug %].
+---
+--- Please log into [% terms.Bugzilla %] and respond as soon as possible
+--- so developers may take action on this [% terms.bug %].
+---
+--- If you have questions on this report, please contact the
+--- [% terms.Bugzilla %] team on https://chat.mozilla.org/#/room/#bmo:mozilla.org.
+---
+--- Thank you,
+--- The [% terms.Bugzilla %] team
```

Comment:
We shouldn't be directing all questions relating to bugs to the Bugzilla team, nor should the Bugzilla team be signing off as the sender.

Both of these presents the request as though it's originating from our team, and that we can answer any question about this; however, this isn't true.  Questions about the request need to be directed as the user setting the needinfo flag, which is already clear in the rest of the content of the email.

Please remove lines 29 through 33.

Reply:
Maybe a page we could link to on how to respond to a needinfo?

Reply:
I put in a place-holder link to documentation on responding to needinfos.